app/components/course-overview.js

```diff
@@ -29,6 +29,14 @@ export default Ember.Component.extend({
       promise: deferred.promise
     });
   }.property(),
+
+  externalIdValidations: {
+    'buffer': {
```

Comment:
The validation criteria, such as `externalIdValidations` need to be passed into any `inplace-{name}` to work. In all five components, it's watching the `buffer` to validate it. Maybe we can talk more about it in person on Monday? I am not how to use `oneWay` on the mixin in this case.